pkg/cmdutil/usage_func.go

```diff
@@ -0,0 +1,226 @@
+package cmdutil
```

Comment:
Many of these functions below seems to be really close to what we did in `pkg/cmd/root/help.go`, maybe we could re-use some of those?

Reply:
Thanks! I used `usage_func.go` to refactor `rootUsageFunc` function at `help.go`